app/controllers/placements/schools/placements_controller.rb

```diff
@@ -12,6 +12,17 @@ def show; end
 
   def remove; end
 
+  def edit_provider
+    @providers = Provider.all
+  end
+
+  def update
+    @placement.provider = provider_params[:provider_name].present? ? Provider.find(provider_params[:provider_id]) : nil
```

Comment:
```suggestion
    @placement.provider = provider_params[:provider_name].present? && provider_params[:provider_id].present? ? Provider.find(provider_params[:provider_id]) : nil
```

Reply:
The ID is funky here, the name is correct as it doesn't come through when deleting text. Looks wrong but it's correct.

I expect this to change/improve once designs come through

Reply:
I realise I could still check the presence here, it's always present with the component but that doesn't stop me testing, I have made a similar change in the second PR I have up as I moved this logic.